tests/integration/cocoon/test_seed_sample_data.py

```diff
@@ -0,0 +1,102 @@
+import unittest
+from pathlib import Path
+import pandas as pd
+import lusid
+from lusidtools.cocoon.seed_sample_data import seed_sample_data
+from lusidtools.cocoon.utilities import create_scope_id
+import logging
+import os
+import datetime
+import pytz
+import random
+
+logger = logging.getLogger()
+
+
+class CocoonTestPortfolioGroup(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+
+        cls.scope = create_scope_id()
+        secrets_file = Path(__file__).parent.parent.parent.joinpath("secrets.json")
+        cls.api_factory = lusid.utilities.ApiClientFactory(
+            api_secrets_filename=secrets_file
+        )
+        cls.sample_data = pd.read_csv(
+            os.path.join("data", "seed_sample_data", "sample_data.csv")
+        )
+
+        seed_sample_data(
```

Comment:
add a test for supplying the override mapping too, you can probably parameterise the tests below

Reply:
Done.

Reply:
Done.